english/play-some-cricket/playsomecricket.md

```diff
@@ -4,3 +4,4 @@
 
 [Mr. Cool Dhoni finishes off.](https://youtu.be/wbfv1qOG3WM)
 
+[Follow cricket news](~/english/cricket/cricket.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.